stl/inc/yvals_core.h

```diff
@@ -808,6 +824,7 @@
 // warning C5278: adding a specialization for 'type trait' has undefined behavior
 // warning C5280: a static operator '()' requires at least '/std:c++23preview'
 // warning C5281: a static lambda requires at least '/std:c++23preview'
+// warning C5285: cannot declare a specialization for 'meow'

```

Comment:
Looks like that we need to resolve the conflict here.

Reply:
I'm working on this now.